packages/eslint-plugin/docs/rules/ban-types.md

```diff
@@ -1,3 +1,7 @@
+---
+displayed_sidebar: someSidebar
```

Comment:
I suppose we should probably rename it to `rulesSidebar` in `sidebar.rules.js`. But that can be a followup issue after this PR IMO.

Reply:
Let's do that as part of this PR.

Reply:
Renamed to rulesSidebar.
dd2ff2d